README.md

```diff
@@ -61,6 +61,9 @@ jenkins ALL=(ALL) NOPASSWD: /usr/bin/docker
 EOF
 ```
 
+docker need to be configured by command:
```

Comment:
I am closing the PR, because there is a conflict with test plugin-path-1 and until it is not fixed we have to go with other solution.